kubernetes/schema_volume_source.go

```diff
@@ -25,6 +26,13 @@ func commonVolumeSources() map[string]*schema.Schema {
 						Description: "Path of the directory on the host. More info: http://kubernetes.io/docs/user-guide/volumes#hostpath",
 						Optional:    true,
 					},
+					"type": {
+						Type:         schema.TypeString,
+						Description:  "Type for HostPath Volume. Defaults to \"\" More info: https://kubernetes.io/docs/concepts/storage/volumes#hostpath",
+						Optional:     true,
+						Default:      "",
```

Comment:
This is the default value for `schema.TypeString` already. Terraform will automatically set it, so this line is not needed.